### PR TITLE
lkft-android: add support for the db845c device

### PR DESCRIPTION
**Pull request by liuyq**
with some changes on the common file.

### CODE REVIEW

File: lava_test_plans/include/fastboot.jinja2 [hunk 1]
```diff
@@ -68,7 +68,6 @@ reboot_to_fastboot: {{ reboot_to_fastboot }}
       minutes: {{ target_deploy_timeout }}
     to: {{ DEPLOY_TARGET }}
     images:
-{% if rootfs == true %}
```
Comment (roxell) [inline lava_test_plans/include/fastboot.jinja2:71]: Please see:
9ad535d7791a ("fastboot: don't download/flash ptable if rootfs is false")

Comment (liuyq) [reply lava_test_plans/include/fastboot.jinja2:71]: what's your suggestion here for the android case that, no rootfs but needs ptable?

Comment (roxell) [reply lava_test_plans/include/fastboot.jinja2:71]: whats your way of booting if its not rootfs?
Can't that be used as a flag too? so 'if rootfs == true or xyz == true'... ?

Comment (liuyq) [reply lava_test_plans/include/fastboot.jinja2:71]: with a new variable it works. 
Anyway I just submitted another PR here, please check there if that works,
https://github.com/Linaro/lava-test-plans/pull/250

Comment (liuyq) [reply lava_test_plans/include/fastboot.jinja2:71]: let's discussion at https://github.com/Linaro/lava-test-plans/pull/250 there, it seems we just need to revert the commit https://github.com/Linaro/lava-test-plans/commit/9ad535d7791aa92d180791931bb79fc08309870a ("fastboot: don't download/flash ptable if rootfs is false") now.

File: lava_test_plans/projects/lkft-android/devices/dragonboard-845c [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{% extends "devices/dragonboard-845c" %}
+
+{% set TAGS = TAGS|default(["lcg"]) %}
+
+{# to avoid overriding by the one definied in testcases #}
+{# like the one definied in testcases/boot.yaml is true by default #}
+{% set auto_login = false %}
```
Comment (roxell) [inline lava_test_plans/projects/lkft-android/devices/dragonboard-845c:7]: Why do you want "auto_login = false" when you do a boot test only?

Comment (liuyq) [reply lava_test_plans/projects/lkft-android/devices/dragonboard-845c:7]: if you check what that auto_login is used for, you would see it's used for the case to login with some username, which is not for the Android case, Android does not need to input the username to login the serial console shell.

**Pull request merged**